docs/variants/novacustom_v560tu/releases.md

```diff
@@ -6,7 +6,78 @@ NovaCustom V560TU
+- [Built-in webcam disable option in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
+- [USB stack disable option in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#usb-configuration)
+- [Network stack disable option in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#networking-options)
+- [Battery threshold options in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#power-management-options)
+- [Intel ME disable option in setup menu](https://docs.dasharo.com/osf-trivia-list/me/)
+- [Block boot when battery is too low](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/359-boot-blocking/#test-cases-common-documentation)
+- [Power on AC option in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#power-management-options)
+- [Keyboard backlight level is restored after suspend or poweroff](https://github.com/Dasharo/dasharo-issues/issues/339)
+- [Fan profiles in setup Menu](https://docs.dasharo.com/unified/novacustom/fan-profiles/)
+- [Fn lock hotkey feature](https://docs.dasharo.com/unified/novacustom/fn-lock-hotkey/)
+- [Throttling temperature adjustment in setup menu](https://docs.dasharo.com/unified/novacustom/features/#cpu-throttling-threshold)
+
+### Known issues
+
+- [No HDMI output in FW on V540TU and V560TU](https://github.com/Dasharo/dasharo-issues/issues/930)
+- [Laggy behaviour on Manjaro (KDE) with open drivers](https://github.com/Dasharo/dasharo-issues/issues/911)
+- [V540TU: Option Previous power state restoration doesn't work](https://github.com/Dasharo/dasharo-issues/issues/931)
+- [Artifacts in video playback in some players using HW acceleration](https://github.com/Dasharo/dasharo-issues/issues/948)
+- [Only native resolution listed for internal panel](https://github.com/Dasharo/dasharo-issues/issues/949)
+
+### Binaries
+
+[novacustom_v56x_mtl_ec_v0.9.0.rom][novacustom_v56x_mtl_ec_v0.9.0.rom_file]{.md-button}
+[sha256][novacustom_v56x_mtl_ec_v0.9.0.rom_hash]{.md-button}
+[sha256.sig][novacustom_v56x_mtl_ec_v0.9.0.rom_sig]{.md-button}
+
+[novacustom_v56x_mtl_v0.9.0.rom][novacustom_v56x_mtl_v0.9.0.rom_file]{.md-button}
+[sha256][novacustom_v56x_mtl_v0.9.0.rom_hash]{.md-button}
+[sha256.sig][novacustom_v56x_mtl_v0.9.0.rom_sig]{.md-button}
+
+[novacustom_v56x_mtl_v0.9.0_dev_signed.rom][novacustom_v56x_mtl_v0.9.0_dev_signed.rom_file]{.md-button}
+[sha256][novacustom_v56x_mtl_v0.9.0_dev_signed.rom_hash]{.md-button}
+[sha256.sig][novacustom_v56x_mtl_v0.9.0_dev_signed.rom_sig]{.md-button}
+
+To verify binary integrity with hash and signature please follow the
+instructions in [Dasharo release signature verification](/guides/signature-verification)
+using [this key](https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/customer-keys/novacustom/novacustom-open-source-firmware-release-0.x-key.asc)
```

Comment:
404

Reply:
fixed in https://github.com/Dasharo/docs/pull/858/commits/ad1a62a1572a679c9b641aae31425891687676fb